### PR TITLE
Typos from Perl5 (1998-2000)

### PR DESCRIPTION
**Pull request by Fourchaux**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -784,6 +785,7 @@ ambigious->ambiguous
 ambigous->ambiguous
 ambiguious->ambiguous
 amendmant->amendment
+amened->amend
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I'd also add amended as a suggestion, personally I'd stick it as the first one.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -824,6 +826,7 @@ analizer->analyzer
 analizes->analyzes
 analizing->analyzing
 analogeous->analogous
+analoguously->analogously
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the non "ly" forms too.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -1156,6 +1159,7 @@ arguements->arguments
 arguemnts->arguments
 argumant->argument
 argumants->arguments
+argumement->argument
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the plural too

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -1574,6 +1578,7 @@ avcoid->avoid
 avcoids->avoids
 avengence->a vengeance
 averageed->averaged
+averloaded->overloaded
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add averload too, and probably the plural

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -2222,6 +2228,7 @@ cauched->caught
 caugt->caught
 cauhgt->caught
 cauing->causing
+causion->caution
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the plural and "ed".

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -2033,6 +2038,7 @@ calculaion->calculation
 calculatable->calculatable, calculable,
 calculatble->calculatable, calculable,
 calculater->calculator
+calculatted->calculated
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add calculatter

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -2547,6 +2554,7 @@ closeing->closing
 clousre->closure
 clsoe->close
 cluase->clause
+clude->clue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add clued as a/the first suggestion.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -2763,6 +2771,7 @@ commom->common
 commond->command
 commongly->commonly
 commonweath->commonwealth
+commpand->command
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add compand as a suggestion

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -3202,6 +3212,7 @@ constrants->constraints
 constrast->contrast
 constrasts->contrasts
 constrcuct->construct
+constrcutor->constructor
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add constrcut

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -3379,6 +3390,7 @@ conveinent->convenient
 conveinience->convenience
 conveinient->convenient
 convenant->covenant
+convenction->convention
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add convection as a suggestion.

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -6531,6 +6558,7 @@ gridles->griddles
 grometry->geometry
 grop->group, drop,
 gropu->group
+gropus->groups
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Possibly add gropes as a suggestion?

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -6986,6 +7015,7 @@ immediatly->immediately
 immediatlye->immediately
 immeditaly->immediately
 immeditately->immediately
+immeidately->immediately
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add immeidate

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -7077,6 +7108,7 @@ implents->implements
 implicite->implicit, implicitly,
 implicitely->implicitly
 implicity->implicitly
+implictly->implicitly
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add implict

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -7155,6 +7187,7 @@ inaugures->inaugurates
 inavlid->invalid
 inbalance->imbalance
 inbalanced->imbalanced
+inbedded->embedded
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add inbed

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -10674,6 +10718,7 @@ privleges->privileges
 privte->private
 prmitive->primitive
 prmitives->primitives
+proably->probably
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add proable

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -10994,6 +11039,7 @@ qoutes->quotes
 qouting->quoting
 qtuie->quite, quiet,
 quadranle->quadrangle
+qualfied->qualified
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add qualfy

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -11329,6 +11375,7 @@ referene->reference
 referenece->reference
 refereneced->referenced
 refereneces->references
+referenence->reference
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the plural and past participle.

File: codespell_lib/data/dictionary.txt [hunk 18]
```diff
@@ -14181,6 +14236,7 @@ unanimuous->unanimous
 unanymous->unanimous
 unaquired->unacquired
 unathorised->unauthorised
+unattained->untended
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Not a typo:
https://www.merriam-webster.com/dictionary/unattained

**Comment by Fourchaux** [comment]
There are still typos to correct?

**Pull request merged**